data.sql

```diff
@@ -1,5 +1,7 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO vet_clinic (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+('Agumon', '2020-02-03', 0, TRUE, 10.23),
+('Gabumon', '2018-11-15', 2, TRUE, 8),
+('Pikachu', '2021-06-07', 1, FALSE, 15.04),
```

Comment:
Great job @IsmaelMastronardi 🤝

- [ ] As stated in the project requirements, kindly ensure you have entered the correct values as provided. You may enter the correct value of `date_of_birth` as '2021/01/07' instead of `2021-06-07`.